api/swagger.yaml

```diff
@@ -6302,6 +6315,50 @@ definitions:
     additionalProperties:
       type: "string"
 
+  PlatformImage:
+    x-nullable: false
+    required: [Id, Descriptor, Available, Platform, ContentSize, UnpackedSize, Containers]
+    description: |
+      PlatformImage represents a platform-specific image that is part of a
+      multi-platform image.
+    type: "object"
+    properties:
+      Id:
+        description: |
+          Content-addressable ID of an image derived from the platform-specific
+          image manifest.
+        type: "string"
+        example: "sha256:95869fbcf224d947ace8d61d0e931d49e31bb7fc67fffbbe9c3198c33aa8e93f"
+      Descriptor:
+        $ref: "#/definitions/OCIDescriptor"
+      Available:
+        description: Indicates whether the image is locally available.
+        type: "boolean"
+        example: true
+      Platform:
+        $ref: "#/definitions/OCIPlatform"
+      ContentSize:
+        description: |
+          The size of the available distributable (possibly compressed) image content
+          in bytes.
+        type: "integer"
+        format: "int64"
+        example: 3987495
+      UnpackedSize:
```

Comment:
+1 on this compromise, I think this is a good way to handle it now w/ the containerd store backend and packed+unpacked, and clients can handle presenting that however it makes most sense.